src/refactor_nrepl/find/find_symbol.clj

```diff
@@ -137,7 +137,16 @@
                                (str/join "/" [namespace var-name]))]
     (->> (core/dirs-on-classpath)
          (mapcat (partial core/find-in-dir (some-fn core/clj-file? core/cljc-file?)))
-         (mapcat (partial find-symbol-in-file fully-qualified-name ignore-errors)))))
+         (shuffle) ;; make it less likely that work will be concentrated in one partition (see next line)
+         (find-util/divide-by (find-util/processor-count))
+         (pmap (fn [work]
+                 ;; `vec` ensures all work is actually performed within the surrounding `pmap`
+                 (->> work
+                      (mapcat (comp vec
+                                    (partial find-symbol-in-file fully-qualified-name ignore-errors)))
+                      vec)))
+         (apply concat)
+         (vec))))
```

Comment:
One might remove this `vec`: art of the point of `pmap` is lazyness.

In `defn find-symbol` at the bottom of this file, `@globals` is the last option in the `or`, so it might make sense to keep it lazy.

OTOH keeping the `(vec)` might be desirable so one can benchmark things without having to remember about seq realisation.

Reply:
> Let's leave it as eager or the value won't be calculated in the background.

Nicely summarised. Didn't realise!